src/translator/service.h

```diff
@@ -144,6 +144,10 @@ class Service {
   std::vector<Response> translateMultiple(std::vector<std::string> &&source,
           TranslationRequest translationRequest);
 
+  /// Returns if model is alignment capable or not. Exists for not breaking things at the extension.
+  // This function, what it does and the name sticks out like a thorn in the entire structure of this class.
+  bool isAlignmentSupported() const { return true; }
```

Comment:
Implement correctly by checking for alignments in the model.

Reply:
`options_` is a member in Service now, and `options_->hasAndNotEmpty("alignment")` is used to provide this. It's untested at the moment. 

Let know if there's something from the model I can use which is stronger.